src/components/ProfilePage/AddressBox/AddressBox.tsx

```diff
@@ -52,12 +52,12 @@ export const AddressBox = ({ address, isEVM, imageLink }: AddressBoxProps) => {
     ? imageLink
     : address && isEVM
       ? `https://effigy.im/a/${address}.png`
```

Comment:
Seems to be duplicated with the above, would be good to have its own component or helper